### PR TITLE
Introduced a System Administration Cheat Sheet

### PR DESCRIPTION
**Pull request by FelicianoTech**
Introduced a sysadmin cheat sheet into the Tools & Reference section. As such, I've also designated 6 articles as featured articles for that section.

### CODE REVIEW

File: docs/tools-reference/sysadmin-cheat-sheet.md [hunk 1]
```diff
@@ -0,0 +1,388 @@
...
+	}
+		#cs-table span.v4note:before {
+			content: "(IPv4)";
+		}
+
+		#cs-table span.v6note:after {
+			content: "(IPv6)"
+		}
+
+#ubuntu:hover {
```
Comment (alexfornuto) [inline docs/tools-reference/sysadmin-cheat-sheet.md]: Nice little Easter Egg

File: docs/tools-reference/sysadmin-cheat-sheet.md [hunk 2]
```diff
@@ -0,0 +1,388 @@
...
+	</tr>
+	<tr>
+		<td>Start in debug mode.</td>
+		<td>
+			service longview stop<br />
+			/etc/init.d/longview debug
+		</td>
+		<td>
+			<strong>CentOS 7 & Fedora 20+</strong><br />
+			systemctl restart systemd-networkd<br />
```
Comment (alexfornuto) [inline docs/tools-reference/sysadmin-cheat-sheet.md]: Pretty sure that's the copy error we were talking about.

Comment (FelicianoTech) [inline docs/tools-reference/sysadmin-cheat-sheet.md]: Pushing a commit shortly to fix it.

**Comment by alexfornuto** [comment]
I've created https://github.com/FelicianoTech/docs/pull/2 which expands your cheat sheet with a "reference sheet" version that's more in the site's style. I've also linked the two, and linked from the sys admin basics guide.

Finally, I created a link reference in your file, to be filled with a PDF version of your table, which you can put in the assets folder.

**Comment by FelicianoTech** [comment]
Any update on this?

**Pull request closed without merging**